test/expressions.ml

```diff
@@ -1376,3 +1441,83 @@ let%expect_test "Lists and Arrays" =
     │ Station 10 ┆ 17    ┆ 13    ┆ 10    ┆ [0.33, 0.67, 1.0]  │
     └────────────┴───────┴───────┴───────┴────────────────────┘ |}]
 ;;
+
+let%expect_test "join_asof" =
```

Comment:
Have you formatted this code with ocamlformat? If not, please install it and run `dune build @fmt` which should apply formatting correctly.

Reply:
I had not formatted the code. Did that now.